support/oidc-discovery-provider/domain_policy_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDomainAllowlist(t *testing.T) {
```

Comment:
can you add a test case for invalid domain?

Reply:
Each subtest has a case for that...

Reply:
Oh, I see what you are saying. SGTM.